modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogCommon.ts

```diff
@@ -6,7 +6,18 @@
  */
 
 import {
-  AlloyEvents, AlloyParts, AlloySpec, AlloyTriggers, Behaviour, DomFactory, GuiFactory, ModalDialog, Reflecting, SystemEvents
+  AlloyComponent,
```

Comment:
Can we please not do this, it's very verbose and just takes up a lot of the room. It should ideally "wrap", instead of "chop" when we have too many imports to fit within 150chars.

- [ ] Fix formatting

Reply:
Not sure if it's wrapped correctly, but I have done so. Is there a way that check can be automated?

Reply:
Not quite correct... if you compare it to the old version, it had it. like this:
```
import {
  ...
} from '@ephox/alloy';
```
whereas your change makes it like this:
```
import { ...
... } from '@ephox/alloy';
```

As far as I'm aware there's no linter available for this 😞

Reply:
Take 2 complete